### ISSUE TITLE
Benchmarks to include in `BLASBenchmarks` docs and `Octavian` docs?

### ISSUE THREAD

**Issue by chriselrod** [chriselrod/BLASBenchmarks.jl#7]
I think size ranges of `1:100` and `logspace(100, 10_000, N)` are good, where `N` is the number of sampled sizes.

I can add benchmarks for Skylake-X, Haswell (probably not all the way up to 10_000, perhaps stopping at just 1_000), and Tiger Lake.

**Comment by DilumAluthge** [comment]
That sounds good to me!

**Comment by chriselrod** [comment]
Depends on how patient you are.

**Comment by DilumAluthge** [comment]
Would be cool if we could use Buildkite to auto generate these benchmark plots.

**Comment by chriselrod** [comment]
Yeah. Now that actions is reenabled, would be good to get around to setting it up.
I've been meaning to upload a few plots to Octavian's README.
